src/js/BookReader/Navbar/Navbar.js

```diff
@@ -21,9 +21,14 @@ export class Navbar {
     const option = this.br.options.controls[controlName];
     if (!option.visible) { return ''; }
     if (option.template) {
-      return option.template(this.br);
+      return `<li>${option.template(this.br)}</li>`;
     }
-    return `<button class="BRicon ${option.className} desktop-only js-tooltip"></button>`;
+    return `<li>
+      <button class="BRicon ${option.className} desktop-only" title="${option.label}">
+        <div class="icon icon-${option.className}"></div>
```

Comment:
suggestion: change div to icon element to be semantic

Reply:
An `<icon>` element doesn't exist in HTML. This is purely presentation, and adds no additional meaning to the UI.